src/guides/getting-started.md

```diff
@@ -573,6 +573,58 @@ That's it! Let's try it out with `cargo run --bin publish_post 1`.
 Published post Diesel demo
 ```
 
+Additionally let's implement a possibility of fetching a single post. We will display the post id with its title.
+Notice `optional()` call. This returns `Option<Post>` instead of throwing an error, which we can then use in our matching pattern.
+
+::: code-block
+
+[src/bin/get_post.rs](https://github.com/diesel-rs/diesel/blob/2.1.x/examples/postgres/getting_started_step_3/src/bin/get_post.rs)
```

Comment:
This should probably link the example from the master branch for now, as the example needs to be merged there.